server/models/user.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+const validate = require('mongoose-validator');
+
+const Schema = mongoose.Schema;
+
+// Email validator
+const emailValidator = validate({
+  validator: 'isEmail',
+  message: 'Provide a proper email address.'
+});
+
+// Create User schema
+const UserSchema = new Schema(
+  {
+    email: {
+      type: String,
+      required: true,
+      validate: emailValidator
+    },
+    password: { type: String, required: true }
+  },
+  { timestamps: true }
```

Comment:
Do we want timestamps?

Reply:
I did add that, wasn't in the issue.  But I think the created_at/updated_at fields can be useful (registration anniversaries, users that haven't updated passwords in x amount of time, etc.)  But can always take out, let me know.

Reply:
Makes sense. (keep)